generated/nidaqmx/nidaqmx_service.cpp

```diff
@@ -72,6 +72,33 @@ namespace nidaqmx_grpc {
     }
   }
 
+  //---------------------------------------------------------------------
+  //---------------------------------------------------------------------
+  ::grpc::Status NiDAQmxService::CalculateReversePolyCoeff(::grpc::ServerContext* context, const CalculateReversePolyCoeffRequest* request, CalculateReversePolyCoeffResponse* response)
+  {
+    if (context->IsCancelled()) {
+      return ::grpc::Status::CANCELLED;
+    }
+    try {
+      auto forward_coeffs = const_cast<const float64*>(request->forward_coeffs().data());
+      uInt32 num_forward_coeffs_in = request->num_forward_coeffs_in();
+      float64 min_val_x = request->min_val_x();
+      float64 max_val_x = request->max_val_x();
+      int32 num_points_to_compute = request->num_points_to_compute();
+      int32 reverse_poly_order = request->reverse_poly_order();
+      response->mutable_reverse_coeffs()->Resize((reverse_poly_order == -1) ? num_forward_coeffs_in : reverse_poly_order + 1, 0);
```

Comment:
Two thoughts:

1. Are we concerned with the client sending -1 for `reverse_poly_order`? I know this can happen with some other sizes passed in but this one _might_ be more likely due to -1 being a sentinel value.
2. Is it fine if the client doesn't set the `reverse_poly_order` field and by default gets 0 so it's sized as one here?

Reply:
Good question. I'm going to change the condition to `< 0` and let the driver report whatever errors it wants to report. I'm OK with the default. There are a lot of cases where we get less-than-awesome defaults.